pyproject.toml

```diff
@@ -1,26 +1,42 @@
-[tool.poetry]
-name = "cits3200-project"
-version = "0.1.0"
-description = "Python packages for the CITS3200 Project"
-package-mode = false
-
-[tool.poetry.dependencies]
-python = "^3.12"
-polars = "^1.3.0"
-loguru = "^0.7.2"
-pytest = "^8.3.2"
-black = "^24.4.2"
-matplotlib = "^3.9.1"
-cytoolz = "^0.12.3"
-ipykernel = "^6.29.5"
-fastexcel = "^0.11.5"
-jupyter-black = "^0.3.4"
-isort = "^5.13.2"
-pathos = "^0.3.2"
-
-[tool.isort]
-profile = "black"
-
-[build-system]
-requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+[tool.poetry]
+name = "cits3200-project"
+version = "0.1.0"
+description = "Python packages for the CITS3200 Project"
+package-mode = false
+
+[tool.poetry.dependencies]
+python = "^3.12"
+polars = "^1.3.0"
+loguru = "^0.7.2"
+pytest = "^7.3.1"

```

Comment:
hm? why is the version different here?

Reply:
I changed pytest version only, based on the comment from Colin

Reply:
Oh so that's manually added and not by `poetry`. Hm I'd say use the newer version unless there's a reason not to, please change it to the newer version